common/test/acceptance/pages/studio/settings.py

```diff
@@ -18,18 +18,28 @@ def is_browser_on_page(self):
         return self.q(css='body.view-settings').present
 
     @property
-    def pre_requisite_course(self):
+    def pre_requisite_course(self, wait_for_field=True):
         """
         Returns the pre-requisite course drop down field.
         """
+        if wait_for_field:
```

Comment:
Can you use [wait_for_element_presence](http://bok-choy.readthedocs.org/en/latest/_modules/bok_choy/page_object.html#PageObject.wait_for_element_presence) here instead?